src/main/java/net/kaczmarzyk/spring/data/jpa/utils/Converter.java

```diff
@@ -77,42 +78,78 @@ public String toString() {
 		}
 	}
 	
-	private static final String DEFAULT_DATE_FORMAT = "yyyy-MM-dd";
-	private static final String DEFAULT_DATE_TIME_FORMAT = "yyyy-MM-dd\'T\'HH:mm:ss";
```

Comment:
Hmmm, isn't it used for `java.util.Date` somewhere? Right now you assigned only "yyyy-MM-dd" to `java.util.Date`. Doesn't it break anything?